### PR TITLE
Upgrade Citus to v11.2.0

### PR DESCRIPTION
**Pull request by xin-hedera**
**Description**:

This PR upgrades Citus to v11.2.0

- bump up citus to v11.2.0
- remove namespace qualifier from citus node hostname

**Related issue(s)**:

Relates to #5409

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: charts/hedera-mirror/templates/configmap-init.yaml [hunk 1]
```diff
@@ -81,13 +81,13 @@ data:
...
   {{- if .Values.citus.enabled }}
-  002-citus.sh: |
+  002-citus.sh: |-
     #!/bin/bash
     set -e
 
     function setup_coordinator() {
       name="{{ include "postgresql.primary.fullname" .Subcharts.citus }}"
-      coordinator="${name}-0.${name}-hl.{{ .Release.Namespace }}.svc.cluster.local"
+      coordinator="${name}-0.${name}-hl"
```
Comment (xin-hedera) [inline charts/hedera-mirror/templates/configmap-init.yaml:90]: remove the namespace suffix so the hostname always points to the pod in the same namespace. this is to fix the issue that when restoring from a previous backup to a different namespace in the same k8s cluster, the citus pods may accidentally connect to the nodes in a different namespace

File: hedera-mirror-importer/src/main/resources/db/scripts/v2/docker/alpine/Dockerfile [hunk 2]
```diff
@@ -4,7 +4,7 @@ FROM postgres:14.5-alpine
 ARG PG_CRON_DB='mirror_node'
 ARG PG_CRON_VERSION=1.4.2
 ARG PG_PARTMAN_VERSION=4.7.1
```
[the remark this answers is not in the document]
Comment (xin-hedera) [reply hedera-mirror-importer/src/main/resources/db/scripts/v2/docker/alpine/Dockerfile:6]: sure. i'd need to rebuild the images too

**Review by steven-sheehy — APPROVED** [review]
LGTM

**Review by jnels124 — COMMENTED** [review]
LGTM

**Review by jnels124 — APPROVED** [review]
LGTM

**Review by edwin-greene — APPROVED** [review]
Looks good

**Pull request merged**